tests/model_fields/test_generatedfield.py

```diff
@@ -94,12 +94,17 @@ def test_database_default_unsupported(self):
             )
 
     def test_db_persist_required(self):
-        msg = "GeneratedField.db_persist must be True or False."
-        with self.assertRaisesMessage(ValueError, msg):
+        with self.assertRaisesMessage(
+            TypeError,
+            "GeneratedField.__init__() missing 1 required keyword-only argument: "
+            "'db_persist'",
+        ):
             GeneratedField(
                 expression=Lower("name"), output_field=CharField(max_length=255)
             )
-        with self.assertRaisesMessage(ValueError, msg):
+        with self.assertRaisesMessage(
+            ValueError, "GeneratedField.db_persist must be True or False."
+        ):
```

Comment:
I'd leave the previous formatting
```suggestion
        msg = "GeneratedField.db_persist must be True or False."
        with self.assertRaisesMessage(ValueError, msg):
```